<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Boja</class>
 <widget class="QDialog" name="Boja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boja</string>
  </property>
  <widget class="QLabel" name="labelSifra">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sifra</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNaziv">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Naziv</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpis">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Opis</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSifra">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNaziv">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOpis">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpisi">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Upisi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonObrisi">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Obrisi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzmeni">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Izmeni</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzadji">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Izadji</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>360</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
